compiler/plc_driver/src/pipelines.rs

```diff
@@ -398,12 +398,12 @@ fn ensure_compile_dirs(targets: &[Target], compile_directory: &Path) -> Result<(
 /// A project that has been transformed into a binary representation
 /// Can be linked to generate a usable application
 #[derive(Debug)]
-pub struct GeneratedProject<'ctx> {
-    target: &'ctx Target,
+pub struct GeneratedProject {
```

Comment:
Just curious, why did you remove the lifetime here?

Reply:
It's not just the lifetime, the target is now owned by the project.
I am not sure but I think this was part of another experiment.
I did change the way we test the integration and I thought i needed a target there, but I could be wrong, i'll try to remove it and see it still works.